### PR TITLE
update text of section 4.10 figure 9 to use floating dates

### PR DESCRIPTION
**Pull request by kdenhartog**
This partially addresses #733 by updating the text to use floating dates. The SVG (claim-example-2.svg) also needs to be updated to further exemplify this. I want to do this as a part of the larger work around SVGs, so updating this only for now.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/806.html" title="Last updated on Aug 31, 2021, 12:18 AM UTC (3f288c6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/806/ea2a3ab...3f288c6.html" title="Last updated on Aug 31, 2021, 12:18 AM UTC (3f288c6)">Diff</a>

### CODE REVIEW

**Comment by kdenhartog** [comment]
The SVG's have been updated by @chaals in #807 now

File: index.html [hunk 1]
```diff
@@ -2018,9 +2018,9 @@ <h4>Presentations Using Derived Credentials</h4>
         <figure>
           <img style="margin: auto; display: block; width: 50%;"
             src="diagrams/claim-example-2.svg" alt="Pat has a property
-            dateOfBirth whose value is 2010-01-01T19:23:24Z">
+            dateOfBirth whose value is 2010-01-01">
           <figcaption style="text-align: center;">
-A basic claim expressing that Pat's date of birth is 2010-01-01T19:23:24Z. Date
+A basic claim expressing that Pat's date of birth is January 1st, 2010. Date
```
Comment (TallTed) [inline index.html:2023]:
```suggestion
A basic claim expressing that Pat's date of birth is January 1, 2010. Date
```

**Review by msporny — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-09-08-vcwg#section3-2) on 2021-09-08

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.2. update text of section 4.10 figure 9 to use floating dates (pr vc-data-model#806)

_See github pull request [#806](https://github.com/w3c/vc-data-model/pull/806)._




**Brent Zundel:** With that, we can move onto PR 806  
… This is updating the text... to use floating dates  
… It's the diagram that's been changed - just something that needs review.  
… I urge folks to check it out and give their approvals on that.  
… Any questions on that before we move forward?  

</details>

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**